JSTests/wasm/stress/cc-int-to-int-tail-call.js

```diff
@@ -1,4 +1,4 @@
-//@ skip
+//@skip
```

Comment:
Accident?

Reply:
I think tail-call is not implemented correctly yet, so we should keep it skip.

Reply:
Unrelated: Did Keith's follow-up fix fix the tail call OMG tests?

Reply:
Ostensibly tail calls should work now